linux_os/guide/auditing/auditd_configure_rules/file_ownership_var_log_audit_stig/oval/shared.xml

```diff
@@ -28,7 +28,7 @@
     <filter action="include">state_owner_not_root_var_log_audit</filter>
   </unix:file_object>
   
-  {{% if product in ["ol8", "rhel8"] %}}
+  {{% if "ol" in product or "rhel" in product %}}
```

Comment:
Normally OL uses `"ol" in families`.

See #12840

Reply:
changed